core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -644,11 +644,8 @@ protected Void visitDescribeInput(DescribeInput node, Integer indent)
         @Override
         protected Void visitQuery(Query node, Integer indent)
         {
-            if (!node.getFunctions().isEmpty() || !node.getSessionProperties().isEmpty()) {
-                builder.append("WITH\n");
-                if (!node.getSessionProperties().isEmpty()) {
-                    append(indent + 1, "SESSION\n");
-                }
+            if (!node.getSessionProperties().isEmpty()) {
+                builder.append("WITH SESSION ");
```

Comment:
Add \n

Reply:
This looks odd when I did that

Reply:
It will look odd for multiple properties, otherwise. We may need to distinguish the two cases:

```
WITH SESSION a = 1
...
```

vs

```
WITH SESSION
    a = 1,
    b = 2
....
```